packages/tokens/scripts/build-parts/getStyleDictionaryConfig.ts

```diff
@@ -5,21 +5,21 @@
 
 import type { Config, DesignToken } from 'style-dictionary/types';
 
-export const targets = ['products', 'devdot', 'marketing', 'cloud-email'];
-export const modes = ['default', 'cds-g0', 'cds-g10', 'cds-g90', 'cds-g100'];
+export const targets = ['products', 'devdot', 'marketing', 'cloud-email'] as const;
+export const modes = ['default', 'cds-g0', 'cds-g10', 'cds-g90', 'cds-g100'] as const;
 
 export type Target = typeof targets[number];
 export type Mode = typeof modes[number];
 
 // uncomment this to enable debugging
```

Comment:
Should this comment be deleted or updated?

Reply:
I'll update it
```suggestion
// comment/uncomment this to disable/enable debugging
```